.github/workflows/ci-approve.yml

```diff
@@ -0,0 +1,33 @@
+# Reference:
+#   - https://github.com/tibdex/github-app-token
+#   - https://github.com/hmarr/auto-approve-action
+
+name: ci-approve
+
+on: pull_request_target
+
+jobs:
+  auto-approve:
+    name: "auto-approve"
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      pull-requests: write
+
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: "generate token"
+        uses: tibdex/github-app-token@v2
+        id: generate-token
+        with:
+          app_id: ${{ secrets.AUTH_APP_ID }}
+          private_key: ${{ secrets.AUTH_APP_PRIVATE_KEY }}
```

Comment:
@tkoyama010 I have a GitHub App that generates tokens on request, so I'm curious to see whether chaining these two actions together works.

This pattern is also adopted in [ci-locks](https://github.com/bjlittle/geovista/blob/main/.github/workflows/ci-locks.yml#L109)